.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27
     - python: 3.6
       env: TOXENV=py36
-    - python: 2.7  # TODO: bump to 3.6
+    - python: 3.6
```

Comment:
IMO not needed any more after that change: https://changelog.travis-ci.com/the-default-python-version-for-your-builds-is-now-3-6-97935

Same for L4.

Reply:
Pls take a look at the updated matrix (that still takes 2.7 into account)

Reply:
`pep8` run under python2.7 and python3.6. I do not know if it is needed, but maybe there are different rules for those versions. It seems to be okey.

Reply:
I put the entire cartesian product of possible envs/versions just to be sure nothing is missed :)